homeassistant/components/ridwell/sensor.py

```diff
@@ -26,8 +26,6 @@
 ATTR_PICKUP_TYPES = "pickup_types"
 ATTR_QUANTITY = "quantity"
 
-DEFAULT_ATTRIBUTION = "Pickup data provided by Ridwell"
```

Comment:
Do we not want to set the attribution at all? If we still want to do that we can implement the `attribution` Entity property.

Reply:
Ah, didn't realize that was a thing! 👍🏻

Reply:
I don't think it's necessary – if Ridwell was getting its data elsewhere, maybe, but since this is the Ridwell integration, I think we're clear. Thoughts?

Reply:
I think it depends on the terms of the integration service provider.